README.md

```diff
@@ -27,10 +27,12 @@ See [a sample report](https://www.fuzzbench.com/reports/sample/index.html).
 
 ## Sample Report
 
-You can view a sample report
+You can view our periodically generated reports
+[here](https://www.fuzzbench.com/reports/index.html) and a sample report
 [here](https://www.fuzzbench.com/reports/sample/index.html).
-This report is generated using 10 fuzzers against 24 real-world benchmarks,
+These reports are generated using 10 fuzzers against 24 real-world benchmarks,
 with 20 trials each and over a duration of 24 hours.
+The raw data in compressed CSV format can be found at the end of each report.
```

Comment:
consider: "can be found" -> "is linked"

Reply:
feels like a nit, can improve in future doc iteration?